src/src/Helpers/AdafruitGFX_helper.cpp

```diff
@@ -1054,7 +1054,8 @@ uint16_t AdaGFXparseColor(String& s, AdaGFXColorDepth colorDepth) {
   }
 
   if ((result == -1) || (result == ADAGFX_WHITE)) {                             // Default & don't convert white
-    if (colorDepth >= AdaGFXColorDepth::Septochrome) {
+    if ((colorDepth >= AdaGFXColorDepth::Septochrome) &&
+        (colorDepth <= AdaGFXColorDepth::Quintochrome)) {
```

Comment:
Just as a remark...
What a stupid color depth naming schema....
It makes it hard to read, and I wonder if the check is correct anyway.
```
if (colorDepth >= 7 && colorDepth <= 5)
```
Not sure if that will ever match.

Reply:
Well, it started with monochrome and duochrome, and after that it got a bit 'weird' 🤣 I've been considering a more comprehendable naming pattern, so I'll change that soon. NB, Quinto = 16.

Reply:
I googled it and the pages suggested by Google all refer to it as "fifth" and variations on it.

I would expect "hexadeca" as 16, bit still I guess "2Tone", or RGB332 , or other more common variants may be more descriptive.

Reply:
I've reworded it to name the used colors/grey-scale to avoid any confusion.